### DISCUSSION TITLE
Are there Usage statistics of devices by groups/users?

### DISCUSSION THREAD

**Discussion by ConfusedMerlin** [DeviceFarmer/stf#559] [Q&A] [answered]
Greetings,

as I did lear today, somebody in the company here made a Java program which shifts through all logs in order to find out how much time which group/user spend with their devices. They using that for monitoring and stuff as it seems, but the method is rather prone for errors, if the source files do not contain all users or stuff like that.

Now I seriously wonder if there is a proper time/usage statistic availalbe somewhere, which can tell what user from what group used what device in a certain range of time. 
If there is any documentation about this already in existence, jus point me to its location; I have proven already that I am able to overlook things written down in pretty big letters in the doc files ;)

**Comment by denis99999** [comment]
@ConfusedMerlin no currently there is no such documentation

**Reply by ConfusedMerlin** [reply]
@denis99999, so no documentation about this topic. Is there an easy way to this kind of information out of the device farm, or are we stuck for now with that log crawling construct that is in use at the moment?

**Comment by denis99999** [comment, marked as answer]
Currently there are no such statistics in the database so indeed it is necessary to go through an external solution such as the exploitation of the logs to obtain them

**Reply by ConfusedMerlin** [reply]
well, that is a bit sad... shall I open a Feature request, or is that kind of feature most outside the scope of future developement?

**Reply by denis99999** [reply]
This feature is well identified but comes from a more general need to have complete statistics on both the client side and the server side, we are thinking about it!
